src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1452,4 +1464,10 @@
   <data name="AvoidAssemblyGetFilesInSingleFileMessage" xml:space="preserve">
     <value>'{0}' will throw for assemblies embedded in a single-file app</value>
   </data>
+  <data name="PlatformCompatabilityCheckUnsupportedOsMessage" xml:space="preserve">
+    <value>'{0}' is unsupported on '{1}'</value>
+  </data>
+  <data name="PlatformCompatibilityCheckSupportedOsMessage" xml:space="preserve">
+    <value>'{0}' is supported on '{1}'</value>
```

Comment:
I suggest this be "is only supported on" rather than "is supported on".  When seeing the error message come up, it's not immediately obvious what the problem is: just because it's supported on "windows" doesn't inherently mean it's not supported elsewhere.

Reply:
Supported platform can be more than one (though for now, we have added only windows support), also we can name all supported platforms too. Anyway, I agree the message is not perfect and we have an issue for further improvement/discussion https://github.com/dotnet/roslyn-analyzers/issues/4021